### PR TITLE
[WIP]Issue #310 now only add city without tests

### PR DESCRIPTION
**Pull request by NataliiaBidiuk**
### Summary

Add City to a tournament, the version without new tests

_closes #310 

#### Design

- [ ] Web API layer contains as little logic as possible
- [ ] Domain objects use semantic names

#### Perfromance

- [ ] In case of any DB query was changed: attach generated SQL for review
- [ ] Number of DB roundtrips should be as low as possible

#### Unit Tests

- [ ] Naming: Initial state and expected result are properly stated
- [ ] Test follows [AAA](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) pattern.
- [ ] Test is Isolated
- [ ] Mock instances and expected instances do not share references
- [ ] When new property is added all related comparers are updated

### CODE REVIEW

**Comment by volleymanagement-sonarqube** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [TournamentValidationSpecification.cs#L15](https://github.com/VolleyManagement/volley-management/blob/b136c3e2e164bab334d222e31f49f9d40b0f629c/src/VolleyManagement.Backend/VolleyManagement.Domain/TournamentsAggregate/TournamentValidationSpecification.cs#L15): The Cyclomatic Complexity of this method is 11 which is greater than 10 authorized. [rule](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1541)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+namespace VolleyManagement.Data.MsSql.Context.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class initial : DbMigration
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs]: Name?

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.Tournaments", "City", c => c.String(maxLength: 50));
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs]: Issue says field should be named location

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/Specifications/TournamentsStorageSpecification.cs [hunk 3]
```diff
@@ -22,11 +22,15 @@ public bool IsSatisfiedBy(TournamentEntity entity)
             var description = new ExpressionSpecification<TournamentEntity>(
                                 t => t.Description == null
                                   || t.Description.Length <= ValidationConstants.Tournament.MAX_DESCRIPTION_LENGTH);
+            var city = new ExpressionSpecification<TournamentEntity>(
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/Specifications/TournamentsStorageSpecification.cs]: No need to update this in future. But you can leave this code here. It won't hurt

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs (hunk 2 above)
Comment (NataliiaBidiuk) [reply src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs]: oops, sorry, I will change it

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs (hunk 1 above)
Comment (NataliiaBidiuk) [reply src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs]: is it good Location? for a name this class?

**Pull request closed without merging**